### DISCUSSION TITLE
`@xstate/test` - testing a machine with no events?

### DISCUSSION THREAD

**Discussion by lougreenwood** [statelyai/xstate#2495] [Q&A]
I have a machine which has no events - instead the machine receives a context and then it traverses through states using `always` and conditional guards to eventually arrive at it's final state.

It seems that `@xstate/test` is designed for testing only when events exist on the machine? Is this assumption correct, should I use a different testing pattern for testing the type of machine I described above?

Thanks in advance! 😃

**Comment by lougreenwood** [comment]
Is there anything unclear about my question? Anything I can do to increase my chance of a response?

**Comment by davidkpiano** [comment]
Can you share an example of the machine you're testing?
